packages/base-map/src/index.tsx

```diff
@@ -190,4 +190,6 @@ const LayerWrapper = (props: LayerProps): JSX.Element => {
   return <>{visible && children}</>;
 };
 
+export const Popup = Styled.Popup;
```

Comment:
I'm not seeing this in the `stop-overlay` or `vehicle-rental-overlay`

Reply:
These packages need a new version number before they can use the popup with the new style (see `index.tsx` in this PR from the `base-map` package).

Reply:
Updated in https://github.com/opentripplanner/otp-ui/pull/471/commits/96fe48e52e433946a26d2b7199136f20574e3443 for storybook viewing only.